src/config/index.ts

```diff
@@ -285,6 +289,17 @@ export const getRateLimit = () => {
   return config.rateLimit;
 };
 
+export const getCacheConfig = () => {
+  const config = loadFullConfiguration();
+  return (
+    config.cache || {
+      maxSizeGB: 2,
+      maxRepositories: 50,
+      cacheDir: './.remote/cache',
+    }
```

Comment:
How does it work if in my config I only set `maxSizeGB`?
Do the other values end up unset?
Do we need to do an object merge to have defaults appear or is that handled further up in the config?